sympy/stats/crv_types.py

```diff
@@ -1177,7 +1177,14 @@ def check(a, b):
     def pdf(self, x):
         a, b = self.a, self.b
         return b**a/gamma(a) * x**(-a-1) * exp(-b/x)
-
+    def sample(self):
+	try :
+	     from scipy.stats import invgamma
+	except ImportError:
+	     raise ImportError("can't import invgamma from scipy.stats")
+	inv_gamma= invgamma.rvs(float(self.a),0,float(self.b))
```

Comment:
There's no need to assign this to a variable if you're just going to immediately return it. You can just write

```py
return invgamma.rvs(float(self.a), 0, float(self.b))
```